esp/bindings/http/platform/httpservice.cpp

```diff
@@ -433,6 +447,12 @@ int CEspHttpServer::processRequest()
             if (authenticateOptionalFailed(*ctx,thebinding))
                 throw createEspHttpException(401,"Unauthorized Access","Unauthorized Access");
 
+
+            if(strieq(method.str(), OPTIONS_METHOD))
+                return onOptions();
```

Comment:
~~This should come after to cors test?  (otherwise a cross site options request will fail?)~~  I see you add it to the options below.